server/src/main/java/org/apache/iotdb/db/qp/physical/crud/InsertRowPlan.java

```diff
@@ -582,6 +582,12 @@ public void recoverFromFailure() {
   @Override
   public void checkIntegrity() throws QueryProcessException {
     super.checkIntegrity();
+    if (this.measurements.length == 1) {
```

Comment:
confusing..
how about if the measurements.length >1?
e.g., `insert into (time,  time) values ...  `
how to understand the second `time`?

Reply:
emm..
thanks for your advise
it seems need to judge measurements only contain one time column or not